server.js

```diff
@@ -265,6 +267,96 @@ cache(function (data, match, sendBadge, request) {
   });
 }));
 
+// PHP version from .travis.yml
+camp.route(/^\/travis(?:-ci)?\/php-v\/([^/]+\/[^/]+)(?:\/([^/]+))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  const userRepo = match[1];  // eg, espadrine/sc
+  const version = match[2] ? match[2] : 'master';
+  const format = match[3];
+  const options = {
+    method: 'GET',
+    uri: 'https://api.travis-ci.org/repos/' + userRepo + '/branches/' + version,
+  };
+  const badgeData = getBadgeData('PHP', data);
+  // Custom user-agent and accept headers are required
+  // http://developer.github.com/v3/#user-agent-required
+  // https://developer.github.com/v3/licenses/
+  const customHeaders = {
+      'User-Agent': 'Shields.io',
+      'Accept': 'application/vnd.github.drax-preview+json'
+  };
+  // require PHP releases
+  regularUpdate(
+    'https://api.github.com/repos/php/php-src/git/refs/tags',
+    (24 * 3600 * 1000), // 1 day
+    tags => {
+      return JSON.parse(tags).
+        // only releases
+        filter((tag) => tag.ref.match(/^refs\/tags\/php-\d+\.\d+\.\d+$/) != null).
+        // get minor version of release
+        map((tag) => tag.ref.match(/^refs\/tags\/php-(\d+\.\d+)\.\d+$/)[1]).
+        // remove duplicates
+        filter((value, index, self) => self.indexOf(value) === index)
```

Comment:
Since you need to do this in two places, could you `npm install lodash.uniq` and use that instead?